vars/releaseManagerAnalyser.txt

```diff
@@ -0,0 +1,18 @@
+Given the release manager output then it analyseS the failure if any, and provides
+an environment variable, `DIGESTED_MESSAGE`, with the digested output to the end user.
+
+```
+
+// analyse the release manager build output
+releaseManagerAnalyser(file: 'release-manager.out')
+
+// send an email\slack with the diggested message
+releaseNotification(slackChannel: "${env.SLACK_CHANNEL}",
```

Comment:
Wdyt about combining both actions into one? Could it be error-prone having to remember how to pass the DIGESTED_MESSAGE to the next step?

Not a blocker, though